packages/experimental/RadioGroup/src/Radio/Radio.types.ts

```diff
@@ -0,0 +1,81 @@
+import type { IViewProps } from '@fluentui-react-native/adapters';
+import { TextProps } from '@fluentui-react-native/text';
+import { FontTokens, IForegroundColorTokens, IBackgroundColorTokens, IBorderTokens } from '@fluentui-react-native/tokens';
+import { IPressableProps } from '@fluentui-react-native/pressable';
+import { IFocusable, IPressableHooks } from '@fluentui-react-native/interactive-hooks';
+import { ColorValue } from 'react-native';
+
+export const radioName = 'Radio';
+
+export interface RadioTokens extends FontTokens, IForegroundColorTokens, IBackgroundColorTokens, IBorderTokens {
+  /**
+   * Focus border color
+   */
+  textBorderColor?: ColorValue;
+
+  /**
+   * Indicator border color
+   */
+  radioBorder?: ColorValue;
+
+  /**
+   * Inner circle color when selected
+   */
+  radioFill?: ColorValue;
+
+  /**
+   * Label text color
+   */
+  labelColor?: ColorValue;
```

Comment:
Can we use `color` from font tokens?

Reply:
Are you suggesting using `fontStyles` directly in the styling versus having `labelColor` as a token here?

Reply:
color would come from IColorTokens but yeah, in RN `color` is recognized as the font color prop since that's the prop name used for font color in the `Text` component.

Reply:
Makes sense, fixed it! Thanks!